CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 1.10.1  (2023-04-03)
+### Changed
+- Fix log example file path in unix tarballs
```

Comment:
was this change applied in a different PR?

Reply:
It was: #87